components/02-elements/password-strength/_password-strength.scss

```diff
@@ -1,12 +1,12 @@
-$password-strength__padding      : 10px 15px !default;
-$password-strength__font-size    : $font-size-small !default;
-$password-strength__border-radius: $border-radius !default;
-$password-strength__background   : $gray-lighter !default;
-$password-strength__background--0: $gray-lighter !default;
-$password-strength__background--1: #ffafae !default;
-$password-strength__background--2: #ffd6b3 !default;
-$password-strength__background--3: #c5eeac !default;
-$password-strength__background--4: #81b562 !default;
+$password-strength__padding                : 10px 15px !default;
```

Comment:
mayby better is use our variable `spacer` and `spacer--medium`?

Reply:
true, forgot about this variable, thanks :)